src/js/redux/actions/search/searchFilterActions.js

```diff
@@ -99,6 +99,17 @@ export const updateRecipientLocations = (state) => ({
     location: state
 });
 
+// Program Source Filter
+export const updateFederalAccountComponents = (state) => ({
+    type: 'UPDATE_FEDERAL_ACCOUNT_COMPONENTS',
+    source: state
```

Comment:
could we rename this param something more descriptive? Just helps w/ readability!

Reply:
Renamed to `source`